### PR TITLE
Feature:  Wrap celery around flask app with waitress and add docker compose

### LINKED ISSUE #394: Add wrap flask app with celery & waitress and add docker-compose
## User stories
As a developer, I want to be able to spawn processes in order to run the simulation in a separate process. Additionally, we want to be able to run the flask app on a waitress server for production.

## Acceptance Criteria
- [x] Add dependencies celery, waitress
- [x] Wrap flask app with celery
- [x] Add dockerfile for dev environment
- [x] Add dockerfile for prod environment
- [x] Add dockerfile for celery
- [x] Add dockercompose file
- [x] Update poe commands
- [x] Add docs about ports
- [x] Add docs about commands

### PR DESCRIPTION
**Pull request by antonykamp**
Fixes #394 

*DISCLAIMER*: I'm not a docker expert! I'm happy to learn new stuff 🙃 

This PR adds celery and waitress. Additionally, it wraps the flask app with celery and introduces the production build with waitress.
Therefore I had to create a big docker compose file and update the poe commands.

## PR checklist

- [x] Acceptance criteria fulfilled
- [x] Additional features are tested
- [x] Docs (code, wiki & diagrams) updated
- [x] Breaking changes anounced
- [x] Dev-branch has been merged into local branch to resolve conflicts
- [x] Tests and linter have passed AFTER local merge
- [x] Another dev reviewed and approved

### CODE REVIEW

File: .env.secret.sample [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
-DATABASE_HOST=
-DATABASE_PORT=
\ No newline at end of file
+DATABASE_NAME=postgres
+DATABASE_USER=postgres
+DATABASE_PASSWORD=root
+DATABASE_HOST=localhost
+DATABASE_PORT=5432
+DATABASE_URL=postgresql://postgres:root@db/postgres
+SECRET=dev
```
Comment (instant-sky) [inline .env.secret.sample]: @antonykamp said this should be `SECRET_KEY`?

**Review by instant-sky — COMMENTED** [review]
Amazing work. I don't understand most of it, but talking about it helped alot!

File: docker-compose.yaml
Comment (antonykamp) [inline docker-compose.yaml:1]: I will split this into core, dev and prod compose file :)

Comment (antonykamp) [reply docker-compose.yaml:1]: Will be fixed later

**Review by CR1337 — COMMENTED** [review]
Is here already anything happening here using waitress and celery?
Or is it just configuration for now?

**Comment by antonykamp** [comment]
Yes and no ;) It configures but not uses celery. And it adds a production command using waitress to start a production version of our application.

**Review by CR1337 — APPROVED** [review]
> Yes and no ;) It configures but not uses celery. And it adds a production command using waitress to start a production version of our application.

Ok, then it looks good I think :)

**Pull request merged**